src/include/messenger/messenger.h

```diff
@@ -189,6 +189,14 @@ class Messenger : public common::DedicatedThreadTask {
   /** @return The ConnectionRouter with the specified router_id. Created by ListenForConnection. */
   common::ManagedPointer<ConnectionRouter> GetConnectionRouter(const std::string &router_id);
 
+  void ProcessDefaultMessage(std::string_view &sender_id, std::string_view &msg, uint64_t recv_cb_id) {
```

Comment:
I am not really sure where I am supposed to invoke the callbacks that are stored in the messenger. Let me know if I should do this inside the replication manager or use something else.